website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md

```diff
@@ -1,8 +1,8 @@
-# Migrate to the EE Helm Chart
+# Migrate
 
-If you are using Appsmith Business Edition, with the Helm chart from `helm.appsmith.com`, this document will help you move to the new Helm chart from `helm-ee.appsmith.com`.
+Follow this guide to move from Appsmith Business Edition Helm chart installation(`helm.appsmith.com`) to Business Edition Helm chart(`helm-ee.appsmith.com`). The new `helm-ee.appsmith.com` installation supports Horizontal Pod Auto Scaling (HPA) which allows Appsmith pods to be automatically scaled based on current load.
```

Comment:
This sentence is off. It reads like "move from Business Edition ... to Business Edition".

This doc, is for those who are using Appsmith Business Edition, with the old Helm chart at `helm.appsmith.com`, to move to the new helm chart, at `helm-ee.appsmith.com`. Can we rephrase to reflect that please?

Reply:
I think adding new and old installations doesn't sound good. How about:

> Use the following guide to move to the Business Edition Helm chart v2 (available at `helm-ee.appsmith.com`). This version includes Horizontal Pod Auto Scaling (HPA) capability, which enables Appsmith pods to scale automatically based on the current workload.

Reply:
I think it is important to mention from where they are moving too.  You can modify it a bit as below

> Follow this guide to move the Appsmith Business Edition installation from the Helm chart at `helm.appsmith.com` to the new Helm chart at `helm-ee.appsmith.com`. This installation supports Horizontal Pod Auto Scaling (HPA), which enables Appsmith pods to scale automatically based on the current load.